JASP-Desktop/analysis/analysisform.h

```diff
@@ -149,6 +159,8 @@ private slots:
 	QMap<QString, JASPControlWrapper* >			_controls;
 
 	QVector<JASPControlWrapper*>				_orderedControls;
```

Comment:
Is this then a list *with* names? 
And it would be cleaner to avoid duplicating the type all the time, so:
```
QVector<JASPControlWrapper*>				_orderedControls,
																	_allItems;
...
QSet<JASPControlBase*>						_jaspControlsWithErrorSet,
																_jaspControlsWithWarningSet;
```